frontend/src/components/housing/enquiry/EnquiryIntakeForm.vue

```diff
@@ -234,6 +234,23 @@ async function emailConfirmation(activityId: string) {
   };
   await submissionService.emailConfirmation(emailData);
 }
+
+async function checkActivityIdValidity(event: Event) {
+  const target = event.target as HTMLInputElement;
+  const activityId = target.value;
+  const hexidecimal = parseInt(activityId, 16);
+
+  if (activityId) {
+    if (activityId.length === 8 || hexidecimal) {
```

Comment:
Style: Not sure if the hex length number 8 is used elsewhere, but if it is, it wouldn't be a bad idea to consider having this be a constant value under constants.ts or similar to avoid magic numbering.